packages/block-editor/src/components/block-edit/with-block-bindings-support.js

```diff
@@ -108,9 +72,9 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 			() =>
 				replacePatternOverrideDefaultBindings(
 					name,
-					props.attributes.metadata?.bindings
+					props.attributes?.metadata?.bindings
```

Comment:
It seems `attributes` is always initialized as an empty object [here](https://github.com/WordPress/gutenberg/blob/5c4ff54d2c682e0a86810071be8944be8ee2d5b9/packages/block-editor/src/components/block-edit/edit.js#L55). I guess it doesn't hurt to add the optional check, but I believe it is not strictly needed?

Reply:
It was an actual problem exposed in unit tests where some attributes might be mocked. It prevents any test failures.

Reply:
The issue existed before https://github.com/WordPress/gutenberg/pull/67370/commits/7c4d983ca8a2c3bcfb563a7aabb3b920a66e0fac. It might no longer be an issue, but I would keep it for addintional safety.